src/components/pages/home/elements/WeatherBlock.js

```diff
@@ -1,49 +1,47 @@
 import React, { Component } from 'react';
 import WeatherTitle from './weatherTitle/WeatherTitle';
-import WeathewrAttributes from './weatherAttributes/WeatherAttributes';
+import WeatherAttributes from './weatherAttributes/WeatherAttributes';
 import getMarsWeatherData from '../../../../api/inSightApiClient';
 import '../HomePage.css';
 import PageNavButton from '../../../shared/PageNavButton';
 
 export default class WeatherBlock extends Component {
   constructor(props) {
     super(props);
-    this.state = { weather: this.dummyValues() };
+    this.state = { weather: null };
   }
 
   componentDidMount() {
     this.getData();
   }
 
   getData() {
-    getMarsWeatherData().then(weather => (this.setState({
-      weather: weather.slice(weather.length - 1)[0],
-    })
-    ));
-  }
-
-  dummyValues() {
-    return ({
-      earthDate: null,
-      sol: null,
-      pressure: {avg: null, max: null, min: null},
-      season: null,
-      temp: {avg: null, max: null, min: null},
-      wind: {avg: null, max: null, min: null, dir: null},
-    });
+    getMarsWeatherData()
+      .then(weather => (this.setState({
+        weather: weather.slice(weather.length - 1)[0],
+      })));
   }
 
   render() {
-    console.log(this.state.weather);
-    return (
+    return this.state.weather ? (
       <div className="weather-block">
-        <WeatherTitle MarsSol={this.state.weather.sol} EarthDate={this.state.weather.earthDate} MarsSeason={this.state.weather.season} />
-        <WeathewrAttributes Temp={this.state.weather.temp} Pressure={this.state.weather.pressure} Wind={this.state.weather.wind} />
+        <WeatherTitle
+          marsSol={this.state.weather.sol}
+          earthDate={this.state.weather.earthDate}
+          marsSeason={this.state.weather.season}
+        />
+        <WeatherAttributes
+          temp={this.state.weather.temp}
+          pressure={this.state.weather.pressure}
+          wind={this.state.weather.wind}
+        />
         <PageNavButton
           buttonLink="/weather"
           message="See more weather info"
         />
       </div>
+    ) : (
+      <div>This will be aloading spinner</div>
```

Comment:
I know we're going to delete this anyway, but there should be a space between `a` and `loading` 😁